src/components/staking/Undelegations.vue

```diff
@@ -5,10 +5,11 @@
         undelegations &&
         undelegations.length > 0
     "
+    class="undelegations"
   >
-    <h3 class="tab-header">
-      Pending Undelegations
-    </h3>
+    <h1>
+      Unstaked Validators
```

Comment:
```suggestion
      Unstaking Validators
```

Reply:
Pending Unstaking makes somehow more sense though. Also because you are not necessarily unstaking everything from a validator

Reply:
i know what you mean - but "Pending Unstaking" doesn't work grammatically. I changed it because we are moving away from "delegations / undelegations"...

We could just do "Pending" maybe?

@jrmoreau what do you think?

Reply:
Trying to visualize this. 

If the page or area of the page gives context for the user that they're looking at where their un-staking is done, I think simple "pending" is best. But without any context, I think we need "pending unstaking"